google/ads/google_ads/client.py

```diff
@@ -94,6 +94,12 @@ def load_from_string(cls, yaml_str):
         config_data = config.parse_yaml_document_to_dict(yaml_str)
         kwargs = cls._get_client_kwargs(config_data)
         return cls(**kwargs)
+    @classmethod  
+    def load_from_dict(cls,config_dict):
+        
```

Comment:
No newline needed here, but please add PyDocs for this method similar to the docs found on the other methods. See this as an example: https://github.com/googleads/google-ads-python/blob/master/google/ads/google_ads/client.py#L100